DataLayer/LightsData/LightsData.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "I_LightsData.h"
+
+class LightsData : public I_LightsData
+{
+public:
+    LightsData();
+    virtual ~LightsData();
+
+    /* LightsData Gets */
+    unsigned char getLightStatus() const;
+
+    /* LightsData Sets */
+    void setLightStatus(const unsigned char&);
+
+private:
+    unsigned char lightStatus_;
```

Comment:
How is this done? It's sitll an unsigned char. should be able to do `lightData.brakeLights()` and get a boolean back.